Source/Charts/Components/YAxis.swift

```diff
@@ -177,6 +178,9 @@ open class YAxis: AxisBase
             min = min - 1.0
         }
         
+        _axisMinimum = min
```

Comment:
I don't think we should change this? Because user set this specifically

Reply:
If someone sets the max and the min value causes the min to be greater than max,  then we exchange their values.
So we should change _axisMinimum and _axisMaximum  too

Reply:
Or we just warn the user that max must be greater than min instead of exchange their values

Reply:
OK, Then we have no need to change min and max
those codes can be removed
https://github.com/danielgindi/Charts/blob/9a9edcb44d0d0c00aaaaa371c03393fc624aeb4e/Source/Charts/Components/YAxis.swift#L155-L169

Reply:
The reason the y-axis label disappearing in your previous comment https://github.com/danielgindi/Charts/pull/3650#issuecomment-424949606 is that you set chartView.leftAxis.axisMaximum = -40.0 specifically (the min of the data in this example is -30, so min>max ), what should we do about this situation?

Reply:
oh my bad. min = max * 0.5  and max = min * 1.5 is wrong for negative values. 

If we use min = max - 1 and max = min + 1 scenario, the min max would be correct.

so either we stick to `min = max - 1 and max = min + 1` , or we use `min = max < 0 ? max * 1.5 : max * 0.5 and max = min < 0 ? min * 0.5 : min * 1.5`.

The idea is the same, make sure display properly, like in this case, .axisMaximum = -40.0, and min is -40 * 1.5 = -60, to display:
<img width="375" alt="image" src="https://user-images.githubusercontent.com/4375169/46644908-e9b05780-cbb4-11e8-858d-e2356b5f44b9.png">

Reply:
Got it!